src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -25,8 +25,15 @@ def get_indexer():  # type: ignore
     return indexer
 
 
+@functools.lru_cache(maxsize=10)
+def get_spanner_indexer():  # type: ignore
+    from sentry.sentry_metrics.indexer.cloudspanner import CloudSpannerIndexer
+
+    return CloudSpannerIndexer()
```

Comment:
I think may want to run two spanners in parallel. A very real use case could be to run one on a multi region spanner and one on a single region spanner.